gateway/response/response.go

```diff
@@ -76,6 +79,8 @@ func loadStatusMap() {
 	statusMap[RequestBodyTooLarge] = ResponseStatus{message: RequestBodyTooLarge, code: http.StatusRequestEntityTooLarge}
 	statusMap[InvalidWriteKey] = ResponseStatus{message: InvalidWriteKey, code: http.StatusUnauthorized}
 	statusMap[InvalidJSON] = ResponseStatus{message: InvalidJSON, code: http.StatusBadRequest}
+	statusMap[TooManyRequestsInMemory] = ResponseStatus{message: TooManyRequestsInMemory, code: http.StatusServiceUnavailable}
```

Comment:
```suggestion
	statusMap[TooManyRequestsInMemory] = ResponseStatus{message: http.StatusText(http.StatusServiceUnavailable), code: http.StatusServiceUnavailable}
```

Reply:
I didn't know we have `http.StatusText()`, this makes use of `statusMap[]` redundant here. So, removed it.